src/UsgsAstroFramePlugin.cpp

```diff
@@ -537,9 +578,14 @@ bool UsgsAstroFramePlugin::canISDBeConvertedToModelState(const csm::Isd &imageSu
       convertible = false;
   }
 
+  csm::Isd localImageSupportData = imageSupportData; 
+  if (imageSupportData.parameters().size() <= 0) {
```

Comment:
This check should use multimap's empty() method instead of checking the size.

Reply:
:+1: